### ISSUE TITLE
checkin reader-intro used for learning how to use AT

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#143]
Currently, a11y qa issue templates include a link to https://bayes.colorado.edu/dev/html/jg-tests/reader-intro.html. I think this is a nice document for teaching screen reader use and I think it should be checked into github instead of hidden in phet-dev.

I was thinking either in the qa repo or phet-info, but a11y-research could also be a good place for it. What do you think @jessegreenberg?

**Comment by lmulhall-phet** [comment]
Is there a downside to having it in multiple places? Does GitHub discourage that sort of thing?

**Comment by zepumph** [comment]
I don't think it matters where it is; I think it only needs one spot, right now it isn't anywhere on github. Do you have a preference?

**Comment by lmulhall-phet** [comment]
I would prefer to have it in the QA repo.

**Comment by jessegreenberg** [comment]
Good idea @zepumph. I agree that it would be best in one spot so that we don't have to update a few copies of documentation.  Ill add it here, maybe in a new directory like `accessibility`, or `doc`?

**Comment by jessegreenberg** [comment]
Added, @zepumph and @lmulhall-phet could you please review? Feel free to add/update this content or move it somewhere else.

**Comment by lmulhall-phet** [comment]
Looks good to me @jessegreenberg!
